File: Data Structures & Algorithms/is-anagram/submission-0.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {

        string sort_s = s;
        sort(sort_s.begin(), sort_s.end());

        string sort_t = t;
        sort(sort_t.begin(), sort_t.end());

        return sort_s == sort_t;
    }
};

File: Data Structures & Algorithms/k-closest-points-to-origin/submission-0.py
class Solution:
    def kClosest(self, points: List[List[int]], k: int) -> List[List[int]]:
        res = []

        points.sort(key = lambda p: math.sqrt(p[0]**2 + p[1]**2))

        for i in range(k):
            res.append(points[i])

        return res